main.py

```diff
@@ -319,82 +354,54 @@ def main():
     parser.add_argument('--filename', help="Path to file where bootimage data can be recorded; will allow for faster execution if script is run multiple times", default="deleted_images.json")
     args = parser.parse_args()
 
+    global DRY_RUN
+    DRY_RUN = False
+    image_list = {}
+    deleted_images_json = {}
+
+    if args.dry_run:
+        DRY_RUN = True
     if args.debug:
         logging.basicConfig(level=logging.DEBUG)
 
     if args.filename:
         deleted_images_filename = args.filename
 
-    ### testing functions
-    # get aliyun images in the installer
-    bootimages = parse_openshift_installer(args.release)
-
-    # get builds with aliyun uploads from a builds.json
-    aliyun_releases = parse_release(args.release)
-
     # preload images that should be deleted
-    deleted_images_json = {}
```

Comment:
I think we need to keep this here for the case of the first time the script is run and there is no JSON file that exists.

Reply:
Oh, nevermind, I see it on line 360